Assignment6_docker/rolling_avg.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/sh
+
+# 1. for sql shell scripting, took reference from stackoverflow
+#    https://stackoverflow.com/questions/7616520/how-to-execute-a-sql-script-from-bash
+#2. using sleep, to start the bda696_mariadb service completely.
+#   Otherwise you might get error "can not connect mysql_service".
+#   Ref https://stackoverflow.com/questions/31746182/docker-compose-wait-for-container-x-before-starting-y
+#3. Reusing homework2.sql with little changes.
+
+
+sleep 5
```

Comment:
I think you do not have to wait for 5s because you have a dependency in the docker-compose file which should guarantee that bda696_mariadb was built and running before the other container started to build.

Reply:
The sleep is a good tactic. The dependency that's set in the docker-compose just checks that the image is running, it knows nothing about if it's actually started up the DB yet or not.